packages/server/src/static/email_templates/_grant_detail.html

```diff
@@ -31,7 +31,7 @@
                                                 style="Margin:0;padding-bottom:16px;">
                                                 <p
                                                     style="Margin:0;-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;font-family:arial, 'helvetica neue', helvetica, sans-serif;line-height:24px;color:#000000;font-size:16px">
-                                                    {{{description}}}<span>... View on <a style="color:#0068D6;text-decoration:none;" href="{{link_url}}">Grants.gov</a></span>
+                                                    {{{description}}}<span>... View on <a style="color:#0068D6;text-decoration:none;" href="{{link_url}}">{{ link_description }}</a></span>
```

Comment:
Noticed that the `href` values are being URL-encoded...
```suggestion
                                                    {{{description}}}<span>... View on <a style="color:#0068D6;text-decoration:none;" href="{{& link_url }}">{{ link_description }}</a></span>
```
Would you mind updating this file to include `&` on any other `href` values provided by mustache?

Reply:
Good catch! 

I'm trying to think of a good way to prevent opening ourselves up to an injection attack here. (I think email clients by and large have a lot of guardrails to prevent stray scripts anyway, but I'd still prefer to avoid the issue on our end.) I think for now I'll adopt the convention of appending `_safe` to a variable name to indicate it's known to be safe, and escape the value from the DB when building the template variables. But if you have another preference here, let me know!

I'm noting we have the same tougher-to-solve issue with rendering the grant description un-escaped here as well, in addition to our use of `v-html` in frontend code. I've added a note to the ticket tracking that issue: https://github.com/usdigitalresponse/usdr-gost/issues/2572#issuecomment-1986096350

Reply:
This solution looks good to me!